### PR TITLE
Adding django-json-widget

### PR DESCRIPTION
**Pull request by swainn**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by swainn** [comment]
I agree to be a maintainer

**Comment by swainn** [comment]
@conda-forge/help-python this is ready to go.

**Comment by swainn** [comment]
@conda-forge/help-python this has been ready for review for over a week.

File: recipes/django_json_widget/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+
```
Comment (BastianZim) [inline recipes/django_json_widget/meta.yaml:1]:
```suggestion
```

File: recipes/django_json_widget/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ec948a4c379687f9c854748ba540fdbeb1b1730846ca953f186818521de7e696
+build:
```
Comment (BastianZim) [inline recipes/django_json_widget/meta.yaml:12]:
```suggestion

build:
```

File: recipes/django_json_widget/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```
Comment (BastianZim) [inline recipes/django_json_widget/meta.yaml:14]: Can this be noarch?

Ref: https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

File: recipes/django_json_widget/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    - python
+    - pip
+  run:
+    - python
+    - future
+    - django >=2.2.13
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
```
Comment (BastianZim) [inline recipes/django_json_widget/meta.yaml:27]:
```suggestion
```

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just a couple of small things, then this is ready to go 🚀

File: recipes/django_json_widget/meta.yaml (hunk 3 above)
Comment (swainn) [reply recipes/django_json_widget/meta.yaml:14]: I belive it can be. I'll update.

**Comment by swainn** [comment]
@BastianZim @conda-forge/python I've addressed the feedback.

**Pull request merged**